app/templates/view_supplier_services.html

```diff
@@ -40,7 +40,11 @@
     field_headings_visible=True
   ) %}
     {% call summary.row() %}
-      {{ summary.service_link(item.serviceName, '/' + item.frameworkFramework + '/services/' + item.id|string) }}
+      {% if item.serviceName %}
+        {{ summary.service_link(item.serviceName, '/' + item.frameworkFramework + '/services/' + item.id|string) }}
+      {% else %}
+        {{ summary.text(item.frameworkName + ' - ' + item.lotName) }}
```

Comment:
The framework name appears in the third column of the table, so maybe we don't need both here?  "Digital Outcomes and Specialists 2 - User research participants" is pretty long and both pieces of information are repeated later in the row.  Probably a question for @laurenceberry I guess.

(Also, this list page is going to be split out into per-framework tables in a story we just wrote today, so including the framework name here will be even less useful then.)

Reply:
Done